Podfile

```diff
@@ -11,5 +11,5 @@ target 'Decred Wallet' do
   pod 'QRCodeReader.swift'
   pod 'Fabric'
   pod 'Crashlytics'
-
+  
```

Comment:
Uhm, I'm a fan of keeping unintended changes out of PRs however small. There's an added space here.

Reply:
You asked that I restore it here: https://github.com/raedahgroup/dcrios/pull/524#discussion_r345099772

Reply:
An empty line, yes. This line is not empty, it has a whitespace.

Reply:
resolved in 835d552